service/internal/handler/count.go

```diff
@@ -0,0 +1,42 @@
+package handler
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/kimbellG/packet-filter/service/pkg/coderror"
+	"github.com/kimbellG/packet-filter/service/pkg/codes"
+)
+
+func (h *Handler) GetCount(w http.ResponseWriter, r *http.Request) {
+	HTTPHandlerWithError(h.getCount).Serve(w, r)
+}
+
+func (h *Handler) getCount(w http.ResponseWriter, r *http.Request) error {
+	type Response struct {
```

Comment:
I would rather create another exported package api, and would create a model api.CountResponse and would return defined models to client